upgrading/automated_upgrades.adoc

```diff
@@ -469,6 +469,20 @@ Automated Upgrade] to ensure you are using the latest upgrade playbooks.
 . Ensure the `openshift_deployment_type` parameter in your inventory file is set
 to `openshift-enterprise`.
 
+. Starting with {product-title} 3.9, the {product-title} web console is deployed
+as a pod on masters during upgrade, and the `openshift_web_console_prefix` is
+introduced to deploy the web console with a customized image prefix. The
+`template_service_broker_prefix` is updated to match other components. If you
+are using a customized *docker-registry* for your installation instead of
+*_registry.access.redhat.com_*, you must specify `openshift_web_console_prefix` and
```

Comment:
I'd say "you must explicitly specify values for `those_parameters`..." but :woman_shrugging: